src/database/databasemanager.cpp

```diff
@@ -135,15 +137,18 @@ bool DatabaseManager::getDatabaseConfig(const std::string &config, int32_t &valu
 
 void DatabaseManager::registerDatabaseConfig(const std::string &config, int32_t value) {
 	Database &db = Database::getInstance();
-	std::ostringstream query;
+	std::string query;
 
 	int32_t tmp;
 
 	if (!getDatabaseConfig(config, tmp)) {
-		query << "INSERT INTO `server_config` VALUES (" << db.escapeString(config) << ", '" << value << "')";
+		query = fmt::format("INSERT INTO `server_config` (`config`, `value`, `worldId`) VALUES ({}, {}, {})", db.escapeString(config), value, g_game().worlds()->getId());
```

Comment:
I recommend maintaining columns in table columns order: ```(`worldId`, `config`, `value`)```
```suggestion
		query = fmt::format("INSERT INTO `server_config` (`worldId`, `config`, `value`) VALUES ({}, {}, {})", g_game().worlds()->getId(), db.escapeString(config), value);
```